src/status_im/contexts/chat/messenger/composer/view.cljs

```diff
@@ -1,183 +1,56 @@
-            :accessibility-label      :chat-message-input}]]]
-        [:<>
-         [gradients/view props state animations show-bottom-gradient?]
-         [link-preview/view]
-         [images/images-list]]
-        [:f> actions/view props state animations window-height subscriptions]]]]]))
+(defn input
+  [_ _]
+  (let [default-value (:input-text (rf/sub [:chats/current-chat-input]))]
+    (fn [set-ref theme]
+      [rn/text-input
+       {:ref                      set-ref
+        :on-change-text           handler/change-text
+        :keyboard-appearance      theme
+        :max-font-size-multiplier 1
+        :multiline                true
+        :placeholder              (i18n/label :t/type-something)
+        :placeholder-text-color   (colors/theme-colors colors/neutral-40 colors/neutral-50 theme)
+        :max-length               constants/max-text-size
+        :accessibility-label      :chat-message-input
+        :style                    (style/input-text theme)
+        :default-value            default-value}])))
 
-(defn f-composer
-  [props]
-  (let [theme                   (quo.theme/use-theme)
-        opacity                 (reanimated/use-shared-value 0)
-        window-height           (:height (rn/get-window))
-        background-y            (reanimated/use-shared-value (- window-height))
-        composer-default-height (+ constants/composer-default-height (:bottom (:insets props)))
-        shared-values           (utils/init-shared-values)
-        extra-params            (assoc props
-                                       :window-height window-height
-                                       :opacity       opacity
-                                       :background-y  background-y
-                                       :theme         theme)
-        props                   (utils/init-non-reactive-state)
-        state                   (utils/init-reactive-state)]
-    [rn/view (when platform/ios? {:style {:z-index 1}})
-     [reanimated/view {:style (style/background opacity background-y window-height)}]
-     [reanimated/view {:style (style/blur-container composer-default-height shared-values)}
-      [quo/blur (style/blur-view theme)]]
-     [:f> sheet-component extra-params props state shared-values]]))
-
-(defn composer
-  [props]
-  (let [current-chat-id       (rf/sub [:chats/current-chat-id])
-        able-to-send-message? (rf/sub [:chats/able-to-send-message?])]
-    (when-not (string/blank? current-chat-id)
-      (if able-to-send-message?
-        [:f> f-composer props]
-        [contact-requests.bottom-drawer/view
-         {:contact-id current-chat-id}]))))
+(defn view
+  []
+  (let [theme     (quo.theme/use-theme)
+        bottom    (safe-area/get-bottom)
+        input-ref (rn/use-ref-atom nil)
+        set-ref   (rn/use-callback #(do
+                                      (rf/dispatch [:chat/set-input-ref %])
+                                      (reset! input-ref %)))]
```

Comment:
To improve the readability (and AFAIK the recommended code style) is to use 

```clojure
(fn [param]
  (a param)
  (b param))
```

Instead of:

```clojure
#(do
  (a %)
  (b %))
```